Sources/MessagingInApp/Gist/Managers/Models/Message.swift

```diff
@@ -17,16 +17,16 @@ public class GistProperties {
 }
 
 public class Message {
-    public private(set) var instanceId = UUID().uuidString.lowercased()
+    public let instanceId = UUID().uuidString.lowercased()
     public let queueId: String?
     public let priority: Int?
     public let messageId: String
-    public private(set) var gistProperties: GistProperties
+    public let gistProperties: GistProperties
 
     var properties = [String: Any]()
```

Comment:
```suggestion
        private(set) var properties: [String: Any]()
```

 I don't fully get the benefit of having `properties` when we already have `gistProperties` which gets set from constructor in any case, even if we decide to keep it, lets only allow read access but restrict write access to the class methods.

Reply:
There are a few instances where `properties` is used directly at the moment. While I agree there's a chance we could replace it with `gistProperties`, I would prefer not to make that change now. Instead, I suggest waiting until rest of the changes are complete so we have a clearer picture of the use case and can replace it more confidently and safely.